### ISSUE TITLE
Support Ruby 3.0.3

### ISSUE THREAD

**Issue by jrgriffiniii** [samvera/hydra-editor#200]
Derived from samvera/maintenance#77

**Comment by jrgriffiniii** [comment]
https://github.com/samvera/active_fedora/issues/1470 is still blocked by https://github.com/samvera/ldp/issues/131, however https://github.com/samvera/ldp/compare/issues-131-jrgriffiniii-ruby3.0?expand=1 is started to advance this.

**Comment by jrgriffiniii** [comment]
https://github.com/samvera/ldp/compare/issues-131-jrgriffiniii-ruby3.0?expand=1 is passing locally, however for CircleCI, the following errors are raised:

```bash
#!/bin/bash -eo pipefail
sudo apt-get update && sudo apt-get install libgmp-dev
Get:1 http://security.debian.org/debian-security buster/updates InRelease [65.4 kB]
Get:2 http://deb.debian.org/debian buster InRelease [122 kB]
Get:3 http://deb.debian.org/debian buster-updates InRelease [51.9 kB]
Reading package lists... Done     
E: Repository 'http://security.debian.org/debian-security buster/updates InRelease' changed its 'Suite' value from 'stable' to 'oldstable'
N: This must be accepted explicitly before updates for this repository can be applied. See apt-secure(8) manpage for details.
N: Repository 'http://deb.debian.org/debian buster InRelease' changed its 'Version' value from '10.3' to '10.11'
E: Repository 'http://deb.debian.org/debian buster InRelease' changed its 'Suite' value from 'stable' to 'oldstable'
N: This must be accepted explicitly before updates for this repository can be applied. See apt-secure(8) manpage for details.
E: Repository 'http://deb.debian.org/debian buster-updates InRelease' changed its 'Suite' value from 'stable-updates' to 'oldstable-updates'
N: This must be accepted explicitly before updates for this repository can be applied. See apt-secure(8) manpage for details.

Exited with code exit status 100
CircleCI received exit code 100
```

**Comment by jrgriffiniii** [comment]
https://github.com/samvera/ldp/pull/136 has been opened, and https://github.com/samvera/active_fedora/issues/1470 can be advanced based upon this.

**Comment by jrgriffiniii** [comment]
https://github.com/samvera/active_fedora/compare/issues-1470-jrgriffiniii-ruby3.0 replaces `solr_wrapper` and `fcrepo_wrapper` with `docker-compose` for local development. However, tests are still failing due to the need to upgrade `active-triples`

**Comment by jrgriffiniii** [comment]
https://github.com/samvera-labs/ActiveTriples/pull/6 provides the updates needed for `active-triples` needed for the upgrade.

**Comment by cjcolvar** [comment]
Closed by #216
